### DISCUSSION TITLE
Trigger event when current tab is pressed

### DISCUSSION THREAD

**Discussion by FelixCEriksson** [dotnet/maui#16699] [Q&A]
Hi, 

I am developing a maui shellapp with tabbar as navigation. I want to create a custom action when a tab is pressed. This can be done by adding a Eventhandler to the Navigating property in the AppShell. eg : 

 public AppShell()
    {
        InitializeComponent();
        Navigating += HandleTabClick;
    }

    private void HandleTabClick(object sender, ShellNavigatingEventArgs e)
    {
        // Do something
    }

The problem is that if i press the current open tab the Navigating event wont be fired. Is there any way to create a custom action that is triggered when the current open tab is pressed?

**Comment by Xyncgas** [comment]
In MAUI.Blazor you can simply listen to onclick event of anything on screen

**Comment by mfaizey** [comment]
Hi, were you able to find any solution?

@FelixCEriksson
